doc/user/pim.rst

```diff
@@ -65,17 +65,33 @@ Certain signals have special meanings to *pimd*.
    prefix of group ranges covered. This command is vrf aware, to configure for
    a vrf, enter the vrf submode.
 
+.. clicmd:: ip pim rp keep-alive-timer (1-65535)
+
+   Modify the time out value for a S,G flow from 1-65535 seconds at RP.
+   The normal keepalive period for the KAT(S,G) defaults to 210 seconds.
+   However, at the RP, the keepalive period must be at least the
+   Register_Suppression_Time, or the RP may time out the (S,G) state
+   before the next Null-Register arrives. Thus, the KAT(S,G) is set to
+   max(Keepalive_Period, RP_Keepalive_Period) when a Register-Stop is sent.
+   If chosing a value below 31 seconds be aware that some hardware platforms
```

Comment:
Typo choosing.

Reply:
Thanks @kuldeepkash for pointing it out. I have fixed it.